script/Ciel/Day 8/08_01_CIEL08_1B.txt

```diff
@@ -36,58 +41,64 @@ You still have some time, right?"
 [sha:e17dad8f100186226f0a8582e1007048a8bbe323]{
 -- Page 36, Offset 18354.
 -- 「ああ、それじゃ普通のお茶を一杯」
-"Ah, then just a normal cup of tea."
+"Sure, I'll have a cup of regular tea then."
 }
 [sha:bf261f06e779e688c01781bc97747dd1a98bac12]{
 -- Page 36, Offset 18355.
 -- 「普通、などというお茶はないのですけどね。琥珀、兄さんにも一つ淹れてあげて。チョイスは貴女に任せるわ」
-"Normal, you say, but there exists no such tea. Kohaku, pour a cup for Nii-san. I'll leave the choice of tea up to you."
+"I don't believe there exists such as thing as 'regular' tea.
+ Kohaku, brew another cup for Nii-san.
+ I'll leave the choice of tea up to you."
 }
 [sha:b8a7da85acd40db69d37d2df287d6020d2058e4f]{
 -- Page 36, Offset 18356.
 -- 「かしこまりました。今朝の献立から考えますとほうじ茶などがよろしいかと」
-"Understood. Considering what was served this morning, something like hojicha should do well."
+"Understood. Considering what was on the menu this morning, I think a houjicha should pair nicely."
 }
 [sha:8bd7b42fdf8ef1192fe72a146c43eeac20dae1bb]{
 -- Page 37, Offset 18357.
 -- 琥珀さんは笑顔で厨房に向かった。
-Kohaku-san cheerfully heads toward the kitchen.
+Kohaku-san cheerfully heads for the kitchen.
 }
 [sha:bf787c1e0cad1c52c962cdc4b6c5cb2dd4ac3ea6]{
 -- Page 37, Offset 18358.
 -- こっちは秋葉の対面のソファーに座って、携帯端末でニュースサイトを流し見る。
 // 携帯端末, aka........ a phone.
-As for me, I sit on the sofa opposite to Akiha and scroll through some news sites on my phone.
+// Nah, Shiki rockin that palm pilot
```

Comment:
did we ever get a satisfactory answer as to why he uses this weird ass antiquated term? Is it just some ~2009 period-piece nonsense?

Reply:
![akkoshrug](https://user-images.githubusercontent.com/2476799/193961536-a0f2de9c-71a8-4a12-8b6e-5a66183e4acb.png)